### PR TITLE
Added all 9 accounting related Command and CommandParser Classes, and other modification.

### PR DESCRIPTION
**Pull request by katenhy**

### CODE REVIEW

**Comment by yewwoei** [comment]
@katenhy 

In the future, do use better commit messages! You have around 10 commit messages that are exactly the same but provide no information of value.

**Comment by katenhy** [comment]
> @katenhy
> 
> In the future, do use better commit messages! You have around 10 commit messages that are exactly the same and could be changed to tell us what is going on: e.g. style fixes.

**Comment by katenhy** [comment]
Login check and test will add in later PR. Thank you so much for the review! Its really helpful :)

File: src/main/java/seedu/address/logic/commands/accounting/AddDebtCommand.java [hunk 1]
```diff
@@ -17,35 +20,44 @@
...
+
+    public static final String MESSAGE_SUCCESS = "Debt Request sent: %1$s %2$s";
+    public static final String MESSAGE_NO_SUCH_USER = "Input user not exist.";
+    public static final String MESSAGE_INVALID_AMOUNT = "Input amount is invalid.";
+
+    private final Username debtor;
+    private final Amount amount;
+
+    /**
+     * Creates an AddDebtCommand to add the specified {@code Debt}model.addRestaurant(toAdd);
```
Comment (chelchia) [inline src/main/java/seedu/address/logic/commands/accounting/AddDebtCommand.java]: should remove code from the javadoc

File: src/main/java/seedu/address/logic/commands/accounting/AddDebtCommand.java [hunk 2]
```diff
@@ -17,35 +20,44 @@
...
+        model.commitAddressBook();
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
```
Comment (chelchia) [inline src/main/java/seedu/address/logic/commands/accounting/AddDebtCommand.java]: should remove code from javadoc

Comment (katenhy) [reply src/main/java/seedu/address/logic/commands/accounting/AddDebtCommand.java]: Resolved! Thanks :)

File: src/main/java/seedu/address/model/ModelManager.java [hunk 3]
```diff
@@ -178,6 +181,85 @@ public void logoutUser() {
...
+        }
+        if (currentUser.getDebts().get(count).getDebtor().equals(user)
+                || currentUser.getDebts().get(count).getCreditor().equals(user)) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean matchStatus(DebtId debtId, DebtStatus status) {
```
[the remark this answers is not in the document]
Comment (katenhy) [reply src/main/java/seedu/address/model/ModelManager.java]: Will change in next PR! Thanks!

File: src/main/java/seedu/address/model/ModelManager.java [hunk 4]
```diff
@@ -178,6 +181,85 @@ public void logoutUser() {
...
+            }
+        }
+        if (currentUser.getDebts().get(count).getAmount().equals(amount)) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean matchUser(DebtId debtId, Username user) {
```
[the remark this answers is not in the document]
Comment (katenhy) [reply src/main/java/seedu/address/model/ModelManager.java]: Will change in next PR! Thanks!

**Pull request merged**